src/components/Department/Department.jsx

```diff
@@ -25,7 +26,8 @@ export default class Department extends Component {
   }
 
   getDepartmentInfo = async () => {
-    const { match: { params: { id } } } = this.props || null;
+    // eslint-disable-next-line react/destructuring-assignment
```

Comment:
just remove the rule

Reply:
+
I've fixed your comments and have done some additional changes